htsohm/material_files.py

```diff
@@ -290,31 +292,45 @@ def write_cif_file(file_name, lattice_constants, atom_sites):
         `$(raspa-dir)/structures/cif/(run_id)-(uuid).cif`
 
     """
+    htsohm_dir = os.path.dirname(os.path.dirname(htsohm.__file__))
+    run_dir = os.path.join(htsohm_dir, run_id)
+    material_dir = os.path.join(run_dir, 'pseudo_materials')
+    file_path = os.path.join(material_dir, "%s.yaml" % uuid)
+    with open(file_path) as material_file:
+        material = yaml.load(material_file)
+
```

Comment:
rather than open / load the file here, maybe it would make more sense to just pass it to the method? It would decouple that.